manifests/reverse_proxy.pp

```diff
@@ -54,12 +54,12 @@
     ssl_options            => ['+StdEnvVars', '+ExportCertData', '+FakeBasicAuth'],
     ssl                    => true,
     ssl_proxyengine        => true,
-    ssl_proxy_ca_cert      => $certs::ca_cert,
+    ssl_proxy_ca_cert      => $certs::katello_default_ca_cert,
     ssl_proxy_machine_cert => $certs::foreman_proxy::foreman_proxy_ssl_client_bundle,
     ssl_cert               => $certs::apache::apache_cert,
     ssl_key                => $certs::apache::apache_key,
-    ssl_chain              => $certs::apache::apache_ca_cert,
```

Comment:
I kind of liked the consistency here to rely on https://github.com/theforeman/puppet-certs/blob/9585baa00b9158f8da16c36b76841df10b92fb97/manifests/apache.pp#L70

Reply:
I don't disagree. I think that should come with or after this change https://github.com/theforeman/puppet-certs/pull/492 that moves deployment of it into Apache's class.

Reply:
How about we adopt this change -- https://github.com/theforeman/puppet-certs/pull/497 and then I update this to rely on those two from the Apache class?

Reply:
I like that.